extensions/cpp-docs/src/index.tsx

```diff
@@ -0,0 +1,460 @@
+import { useEffect, useState } from "react";
+import {
+  Action,
+  ActionPanel,
+  Detail,
+  Icon,
+  List,
+  Toast,
+  showToast,
+  getPreferenceValues,
+  closeMainWindow,
+  updateCommandMetadata,
+} from "@raycast/api";
+import { shellEnv } from "shell-env";
+import { exec } from "child_process";
+import { runAppleScript } from "run-applescript";
+import fs from "fs";
+import { promisify } from "util";
+
+interface EnvType {
+  env: Record<string, string>;
+  cwd: string;
+  shell: string;
+}
+
+interface Preferences {
+  arguments_terminal: string;
+  arguments_terminal_type: string;
+  documentation_source: string;
+}
+
```

Comment:
```suggestion
```
You don't need to manually set preference types as this is autogenerated in `raycast-env.d.ts`x